<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wndMateriales</class>
 <widget class="QMainWindow" name="wndMateriales">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Material Library</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
    <item>
     <widget class="QTreeWidget" name="treeMateriales">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="itemsExpandable">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
       <property name="font">
        <font>
         <kerning>true</kerning>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Measure Unit</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tags</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Complex</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="grpDetails">
      <property name="title">
       <string>Details</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>2</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Name</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="txtNombre"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Description</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QTextEdit" name="txtDescripcion">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>120</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Tags</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="txtTags"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Measure Unit</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="cboMeasureUnit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QCheckBox" name="chkUsableDirectly">
              <property name="text">
               <string>Use directly in project.</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCommandLinkButton" name="cmdGoogleIt">
              <property name="text">
               <string>Find in internet</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTreeWidget" name="treeComponentes">
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
            <property name="toolTip">
             <string>Componentes que forman este material</string>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Quantity</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Measure Unit</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Complex</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Materials</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewMaterial"/>
   <addaction name="actionSaveMaterial"/>
   <addaction name="actionSaveLibrary"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionNewMaterial">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/dlg/dock/add</normaloff>:/dlg/dock/add</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New Material</string>
   </property>
  </action>
  <action name="actionSaveLibrary">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/dlg/dock/save</normaloff>:/dlg/dock/save</iconset>
   </property>
   <property name="text">
    <string>SaveLibrary</string>
   </property>
  </action>
  <action name="actionSaveMaterial">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/dlg/dock/ok</normaloff>:/dlg/dock/ok</iconset>
   </property>
   <property name="text">
    <string>Save Material</string>
   </property>
  </action>
  <action name="actionAddComponente">
   <property name="text">
    <string>AddComponente</string>
   </property>
   <property name="toolTip">
    <string>Add Component</string>
   </property>
  </action>
  <action name="actionDeleteComponente">
   <property name="text">
    <string>Remove Component</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
